tests/test_utils/tests.py

```diff
@@ -872,6 +873,40 @@ class MyCustomField(IntegerField):
         self.assertFieldOutput(MyCustomField, {}, {}, empty_value=None)
 
 
+class AssertSignalSentTests(TestCase):
```

Comment:
Defining a `Signal()` instance for testing purpose would be better than relying on model signals IMHO as it would be more explicit. This would also have the nice side effect of not requiring database queries to be run, allowing you to subclass `SimpleTestCase` instead of `TestCase`.